### ISSUE TITLE
Website Licenses page does not work

### ISSUE THREAD

**Issue by Holyphoenix** [LibreWeather/libre-weather#14]
When navigating to the Licenses page on the libreweather website, I am met with 404 error.

https://libreweather.com/licenses

**Comment by TobiTenno** [comment]
Thanks, looking into it!

**Comment by TobiTenno** [comment]
Has it ever worked for you before? I'm wondering if we never actually got it working in prod, but if it worked in dev only

**Comment by Holyphoenix** [comment]
No, I don't think it ever worked in prod. The build script probably needs to be updated to call the build:licenses script to make sure it builds the licenses file and includes it in assets before it compiles the actual app.

**Comment by TobiTenno** [comment]
Hmmm, okiedoke. I thought I had those set up already, but I'll definitely look. Thanks.

**Comment by TobiTenno** [comment]
looks like it was because we're using an SPA but not redirecting traffic through index.html.... peeps over at parcel got me sorted
